src/neptune/exceptions.py

```diff
@@ -790,32 +791,39 @@ def __init__(self, project: QualifiedName):
         super().__init__(message.format(project=project, **STYLES))
 
 
-class NeptuneIntegrationNotInstalledException(NeptuneException):
-    def __init__(self, integration_package_name, framework_name):
+class NeptuneMissingRequirementException(NeptuneException):
+    def __init__(self, package_name: str, framework_name: Optional[str]):
         message = """
-{h1}
-----NeptuneIntegrationNotInstalledException-----------------------------------------
-{end}
-Looks like integration {integration_package_name} wasn't installed.
-To install, run:
-    {bash}pip install {integration_package_name}{end}
-Or:
-    {bash}pip install "neptune[{framework_name}]"{end}
-
-You may also want to check the following docs page:
-    - https://docs.neptune.ai/integrations
-
-{correct}Need help?{end}-> https://docs.neptune.ai/getting_help
-"""
+    {h1}
+    ----{exception}-----------------------------------------
+    {end}
+    Looks like package {package_name} wasn't installed.
+    To install, run:
+        {bash}pip install {package_name}{end}
+    Or:
+        {bash}pip install "neptune[{framework_name}]"{end}
+
+    You may also want to check the following docs page:
+        - https://docs.neptune.ai/integrations
+
+    {correct}Need help?{end}-> https://docs.neptune.ai/getting_help
+    """
+        framework_name = framework_name if framework_name else package_name
         super().__init__(
             message.format(
-                integration_package_name=integration_package_name,
+                exception=self.__class__.__name__,
+                package_name=package_name,
                 framework_name=framework_name,
                 **STYLES,
             )
         )
 
 
+class NeptuneIntegrationNotInstalledException(NeptuneMissingRequirementException):
```

Comment:
@Raalsky tbh I'm not sure I see a need for a separate exception type for integrations. It makes it all more complicated. How about we just throw a `NeptuneMissingRequirementException` and always link a docs page for integrations? Also @normandy7, what do you think?

Reply:
You mean removing the installation instructions from the exception message? What are the pros/cons with this change (from an end-user perspective)? It would be ideal to have an installation command handy for copy-pasting

Reply:
No no, actually it will be just about renaming `NeptuneIntegrationNotInstalledException` to `NeptuneMissingRequirementException` and small tweaks in the message (changing 'integration' to just 'package') to make it more generic, as we'll also probably be checking installation of things like `pandas` which is not an integration per se

Reply:
The rest will stay the same - installation instruction and link to the docs. I just don't see a need to create two separate entities which only differ in names and some small message wording details

Reply:
Oh I see! Yes, that should be fine.

Reply:
I will introduce a simplification shortly and ask you to review, ok? :smiley:

Reply:
Certainly 👍🏻 I'll monitor my inbox

Reply:
There's a lot of files, but it's mostly just updating how integrations are imported. The thing to check is probably only `src/neptune/exceptions.py` :)